.gitignore

```diff
@@ -1,6 +1,3 @@
-# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
```

Comment:
IMHO: It should be: 
- The default github .gitigonore for go https://github.com/github/gitignore/blob/master/Go.gitignore
+
- What is relevant for the project.